sql/GetTotalsForContracts.sql

```diff
@@ -0,0 +1,42 @@
+SELECT
+  t."tokenAddress" AS "tokenAddress",
+  COUNT(t."tokenAddress") AS "holders"
```

Comment:
You might need distinct count here no? Can there be multiple records with same tokenAddress?